fetch.bs

```diff
@@ -7797,15 +7815,24 @@ method steps are:
    <li><p><a for=/>Resolve</a> <var>p</var> with <var>responseObject</var>.
   </ol>
 
+ <li><p>Let <var>error</var> be <var>requestObject</var>'s <a for=Request>signal</a>'s
+ <a for=AbortSignal>abort reason</a> if it not undefined. Otherwise, let <var>error</var> be an
+ "{{AbortError}}" {{DOMException}}.
+
+ <li><p>Let <var>serializedError</var> be the result of calling [$StructuredSerialize$]
```

Comment:
I don't believe this is the right place to serialize. It should be done in `abort fetch`, not when the fetch is started.